src/App.tsx

```diff
@@ -1,9 +1,27 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+// type Movie = {
+//   title: string,
+//   description: string,
+//   imgUrl: string,
+//   imdbUrl: string,
+//   imdbId: string,
+// };
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [moviesToRender, setMoviesToRender] = useState(moviesFromServer);
+
+  useEffect(() => {
+    setMoviesToRender([...moviesFromServer].filter(({ title, description }) => {
```

Comment:
It's better to create a separate function for filtering movies and call it inside useEffect.